### ISSUE TITLE
Build fails

### ISSUE THREAD

**Issue by phil-opp** [gz/rust-x86#18]
The build fails on the latest nightly with the following error:

```
src/segmentation.rs:206:43: 206:50 error: clobber should not be surrounded by braces
src/segmentation.rs:206          1:" :: "r" (sel.bits() as u64) : "{rax}" "memory");
```

**Comment by Ericson2314** [comment]
https://github.com/rust-lang/rust/commit/e32da1275300a437490dd2e37ec49d7f11e3d780

**Comment by phil-opp** [comment]
This issue still affects versions `0.5.0` - `0.7.0` on crates.io (older versions have other errors). Could you release new patch versions for them?

**Comment by gz** [comment]
I published a new version 0.7.1 with your fix.
Note to self: Use git tags for versions from now on ;-/

**Comment by gz** [comment]
Do you or some of your users still use 0.5.x and 0.6.x versions as well?

**Comment by phil-opp** [comment]
I'm not sure about 0.5.x, but I still have 0.6.x in the blog post. I'll update it, but I think a 0.6.x patch would be nice nonetheless

**Comment by gz** [comment]
I published 0.6.1.

**Comment by phil-opp** [comment]
Thanks a lot!
